docker/Dockerfile

```diff
@@ -1,30 +1,26 @@
-FROM golang:1.9-alpine3.7
+FROM golang:1.9-alpine3.7 as builder
+
+RUN apk --no-cache --update add make git musl-dev
 
 RUN mkdir -p /go/src/github.com/signal18/replication-manager
 WORKDIR /go/src/github.com/signal18/replication-manager
 
-RUN mkdir -p \
-        /etc/replication-manager \
-        /usr/share/replication-manager/dashboard \
-        /var/lib/replication-manager 
-
-RUN \
-    apk --no-cache --update add make git musl-dev && \ 
-    rm -rf /var/cache/apk/*
-
 COPY . .
 
 RUN make osc cli
 
-COPY dashboard /usr/share/replication-manager/dashboard/
 
-RUN mv build/binaries/replication-manager-osc /go/bin/replication-manager \
-    && mv build/binaries/replication-manager-cli /go/bin/
+FROM alpine:3.7
+
+RUN mkdir -p \
+        /etc/replication-manager \
+        /var/lib/replication-manager
 
-WORKDIR /go/bin
+COPY --from=builder /go/src/github.com/signal18/replication-manager/dashboard /usr/share/replication-manager/dashboard/
+COPY --from=builder /go/src/github.com/signal18/replication-manager/build/binaries/replication-manager-osc /usr/bin/replication-manager
+COPY --from=builder /go/src/github.com/signal18/replication-manager/build/binaries/replication-manager-cli /usr/bin/replication-manager-cli
 
-RUN rm -rf /go/src /go/pkg && apk --no-cache del make git musl-dev
-RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
+#RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
```

Comment:
I see you have commented this, is there a particular reason for doing it?

Reply:
It's a relic. I wasn't sure whether it's still needed. Turned out it's apparently not.

I've removed the line.